code/game/objects/structures/gate.dm

```diff
@@ -114,7 +137,8 @@ GLOBAL_LIST_EMPTY(biggates)
 	sleep(10)
 	for(var/turf/T in turfsy)
 		for(var/mob/living/M in T)
-			M.gib()
+			M.log_message("has been crushed by the [src]!", LOG_ATTACK)
+			crush(M)
 	density = initial(density)
 	opacity = initial(opacity)
 	layer = initial(layer)
```

Comment:
line 145 and 146 may be causing the gate being opaque when closed . you should add an exception for the new blocker you /obj/gblock/not_opaque you created

Reply:
on line 123 and 124 there is the same but to make them transparent if you change it here change it there aswell